.github/workflows/mdbook_cd.yml

```diff
@@ -23,8 +23,8 @@ jobs:
       - name: Decompress mdbook build artifact 📦
         run: mkdir book && tar -xvf spec_mdbook-${{ github.sha }}/spec_mdbook.tar ./book && ls ./book
 
-      - name: Deploy mdBook to github pages 🚀
-        uses: peaceiris/actions-gh-pages@v3
+      - name: Deploy MDBook to gh-pages 🚀
+        uses: JamesIves/github-pages-deploy-action@4.0.0
         with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          publish_dir: ./book
+          branch: gh-pages
+          folder: ./book/html
```

Comment:
Compare this to https://github.com/LibertyDSNP/spec/blob/3371b7486b7d68e314b052588547a12844c071b5/.github/workflows/deploy.yml#L38